docs/eval-framework.md

```
@@ -0,0 +1,226 @@
+Config                                Prec    Rec     F1   Cons    Time     Cost    Tokens
+------------------------------------------------------------------------------------------
+gemini/tools-off-thinking-high        0.86   0.60   0.71    50% 132.2s    n/a     435.0k
+codex/tools-off-thinking-high         0.80   0.40   0.53    40%  96.4s    n/a      11.6k
+claude/tools-off-thinking-high        0.00   0.00   0.00     0% 124.7s   $0.28     6.8k
+```
+
+### Per-issue detection rates
+
+Below the table, each ground truth issue is listed with its detection rate per adapter, grouped by difficulty:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)